libs/openFrameworks/gl/ofTexture.h

```diff
@@ -161,6 +161,9 @@ class ofTextureData {
 #ifndef TARGET_OPENGLES
 		glInternalFormat = GL_RGB8;
 		textureTarget = GL_TEXTURE_RECTANGLE_ARB;
+#elif defined(TARGET_OPENGLES) && defined(TARGET_OPENGLES_3_0)
+		glInternalFormat = GL_RGB16F;
+		textureTarget = GL_TEXTURE_2D;
```

Comment:
@arturo this part needs to be realtime I believe rather than define. I think I need to lose the new defines for ES3 as well.